libs/perun/components/src/lib/perun-footer/perun-footer.component.html

```diff
@@ -1,92 +1,32 @@
-          </li>
-          <li>
-            <a target="_blank" [href]="administratorDocumentationWebpage"
-               [ngStyle]="{'color': linksTextColor}">{{'SHARED_LIB.PERUN.COMPONENTS.FOOTER.ADMINISTRATOR_DOCUMENTATION' | translate}}</a>
-          </li>
-          <li>
-            <span (click)="openBugReportDialog()" id="clickable"
-               [ngStyle]="{'color': linksTextColor}">{{'SHARED_LIB.PERUN.COMPONENTS.FOOTER.REPORT_ISSUE' | translate}}</span>
-          </li>
-        </ul>
-
-      </div>
-      <!-- Grid column -->
-      <div class="col-md-4 mx-auto footer-col">
-
-        <!-- Links -->
-        <h6 class="text-uppercase font-weight-bold" [ngStyle]="{'color': footerHeadersTextColor}">
-          {{'SHARED_LIB.PERUN.COMPONENTS.FOOTER.SUPPORT' | translate}}
-        </h6>
-        <ul class="list-unstyled">
-          <li [ngStyle]="{'color': linksTextColor}">
-            <i class="material-icons align-bottom">
-              mail
-            </i>
-            {{supportMail}}
-          </li>
-        </ul>
-
+<footer [ngStyle]="{'background': bgColor}" class="wrap-content text-md-left pr-xl-5 pl-xl-5">
+  <div class="row">
+    <div *ngFor="let item of items" class="footer-col col">
+      <h6 [ngStyle]="{'color': headersTextColor}" class="ml-1 font-weight-bold">
+        {{item | localisedText: language:'title'}}
+      </h6>
+      <div *ngIf="item.elements">
+        <div *ngFor="let elem of item.elements" class="ml-1 row row-height">
+          <i *ngIf="elem.icon"  [ngStyle]="{'color': linksTextColor}" class="material-icons align-bottom mr-1">
+            {{elem.icon}}
+          </i>
+          <span (click)="openDialog(elem.dialog)" *ngIf="elem.dialog"
+                [ngStyle]="{'color': linksTextColor}"
+                id="clickable">{{elem | localisedText: language:'label'}}
+          </span>
+          <a *ngIf="elem.link && !elem.dialog" [href]="elem.link" [ngStyle]="{'color': linksTextColor}"
+             target="_blank">{{elem | localisedText: language:'label'}}</a>
+          <p *ngIf="!elem.link && !elem.dialog"
+             [ngStyle]="{'color': linksTextColor}">{{elem | localisedText: language:'label'}}</p>
+        </div>
       </div>
     </div>
-    <!-- Grid row -->
-
   </div>
-  <!-- Footer Links -->
-  <hr class="my-0">
-  <!-- Copyright -->
-  <div class="footer-copyright text-center py-2" id="copyright"
-       [ngStyle]="{'color': footerCopyrightTextColor}">
-    © {{this.currentYear}} Copyright:
-    <span *ngFor="let item of this.copyright">
-      <a target="_blank" [href]="item.webpage"> {{item.name}}</a>,
+  <hr class="my-0 mt-3">
+  <div [ngStyle]="{'color': copyrightTextColor}" class="font-fix text-center">
+    © {{currentYear}} Copyright:
+    <span *ngFor="let item of copyrightItems; let index = index">
+      {{index ? ', ' : '&nbsp;'}}<a [href]="item.url" [ngStyle]="{'color': copyrightTextColor}"  class="text-decoration-none" target="_blank">{{item.name}}</a>
```

Comment:
```suggestion
      <a [href]="item.url" [ngStyle]="{'color': copyrightTextColor}"  class="text-decoration-none" target="_blank">{{index ? ', ' + item.name : item.name}}</a>
```

thix fixes the gap in front of the names